packages/alchemy/src/react/components/navigation.tsx

```diff
@@ -4,12 +4,14 @@ import { Button } from "./button.js";
 interface NavigationProps {
   onBack?: () => void;
   onClose: () => void;
-  showingBack: boolean;
+  showBack: boolean;
```

Comment:
hmmm wouldn't close and back be mutually exclusive here? Would be better to have something like:

`exitButton?: {type: "close" | "back", onExit: () => void}`